wp-content/civi-extensions/civirazorpay/api/v3/Civirazorpay/CivicrmRazorpayFetchSettlementDateCron.php

```
@@ -0,0 +1,323 @@
+    try {
+      $processorConfig = PaymentProcessor::get(FALSE)
+        ->addWhere('payment_processor_type_id:name', '=', 'Razorpay')
+        ->addWhere('is_test', '=', $this->isTest)
+        ->execute()->single();
+
+      $this->processorID = $processorConfig['id'];
+      $this->apiKey = $processorConfig['user_name'];
+      $this->apiSecret = $processorConfig['password'];
+    }
+    catch (Exception $e) {
+      throw new Exception($errorMsg);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$errorMsg'. (Unused Code Rules)

(UnusedLocalVariable)